prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.ts

```diff
@@ -69,9 +70,10 @@ export class LicenseClassesMaintenancePageComponent implements OnInit {
             collegeName: college.name,
             licenseCode: collegeLicense.licenseCode,
             discontinued: collegeLicense.discontinued,
+            collegeLicenseGroupingCode: license.collegeLicenses[0].collegeLicenseGroupingCode,
```

Comment:
Is there a reason this can't be `collegeLicenseGroupingCode: collegeLicense.collegeLicenseGroupingCode,`?  Seems simpler

Reply:
I just double checked. The "collegeLicense" is from college config which does not contain the collegeLicenseGroupingCode value (always null). Only license config contains the collegeLicenseGroupingCode value.

Reply:
Thanks for changing